awswrangler/catalog/_get.py

```diff
@@ -120,6 +121,9 @@ def get_table_types(
     catalog_id
         The ID of the Data Catalog from which to retrieve Databases.
         If ``None`` is provided, the AWS account ID is used by default.
+    return_iceberg_current
+            If True, returns only current iceberg fields (fields marked with iceberg.field.current: true).
+            Otherwise, returns the all fields. False by default (return all fields).
```

Comment:
nit:
```
            If True, returns current Iceberg fields only (i.e. fields marked iceberg.field.current: true).
            Otherwise, returns all fields. False by default (return all fields).
```